toy-project/components/checkfollow/FollowList.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import ListElement from './ListElement';
+
+interface userInfoTypes {
+  id: number;
+  userName: string;
+  imageUrl: string;
+}
+
+/** dummy data */
+const USER_INFO: userInfoTypes[] = [
+  { id: 1, userName: 'aaaaaa', imageUrl: '' },
+  { id: 2, userName: 'bbbbbb', imageUrl: '' },
+  { id: 3, userName: 'cccccc', imageUrl: '' },
+  { id: 4, userName: 'dddddd', imageUrl: '' },
+  { id: 5, userName: 'eeeeee', imageUrl: '' },
+  { id: 6, userName: 'ffffff', imageUrl: '' },
+  { id: 7, userName: 'gggggg', imageUrl: '' },
+  { id: 8, userName: 'hhhhhhh', imageUrl: '' },
+  { id: 9, userName: 'iiiiiii', imageUrl: '' },
+  { id: 10, userName: 'jjjjjj', imageUrl: '' },
+  { id: 11, userName: 'kkkkkk', imageUrl: '' },
+];
+
+export default function FollowList() {
+  return (
+    <section className="commonBackground flex h-[480px] w-full flex-col">
+      <div className="flex justify-end gap-[14px] px-[25px] py-[20px]">
+        <button className="commonButton h-[25px] w-[65px] bg-white text-black hover:bg-red hover:text-white">
```

Comment:
이거를 common.css 햇던 것처럼 아예 class하나로 지정해서 한번에 하는 방법도 있어요! 

```suggestion
        <button className="commonButton buttonTheme">
```